test/optimizations/bulkcomm/bharshbarg/stridedAssignDR.good

```diff
@@ -0,0 +1,45 @@
+stridedAssignDR.chpl:25: In function 'buildSlices':
```

Comment:
This test should be updated to use the new type. I can help you figure out what this program is doing, or I can make the change myself post-merge.

Reply:
I've made an issue for rewriting this here: https://github.com/cray/chapel-private/issues/374